src/main/common/clients/featureTogglesClient.ts

```diff
@@ -1,13 +1,19 @@
 import { request as requestPromiseApi, RequestPromiseAPI } from 'main/app/client/request'
 import { User } from 'main/app/idam/user'
 import * as config from 'config'
+import { trackCustomEvent } from 'logging/customEventTracker'
 
-export const featureTogglesApiBaseUrl: string = `${config.get<string>('feature-toggles-api.url')}`
-const featureTogglesApiUrl: string = `${featureTogglesApiBaseUrl}/api/ff4j/check`
+let featureTogglesApiUrl: string
```

Comment:
This is being overwritten by each instantiation of a `FeatureTogglesClient`. Make this just an instance variable